CHANGELOG.md

```diff
@@ -3236,6 +3236,8 @@ _Changes since `grafana/cortex-jsonnet` `1.9.0`._
    }
    ```
 * [FEATURE] Added multi-zone ingesters and store-gateways support. #1352 #1552
+* [FEATURE] The following alertmanager limit was added: #9402
```

Comment:
This needs to go at the top of the file (and not in the jsonnet section)

Reply:
lol... Tks, done now 👍